app/controllers/my-projects/project/recommendations/add.js

```diff
@@ -127,6 +127,8 @@ export default class MyProjectsProjectRecommendationsAddController extends Contr
       });
       disposition.save();
     });
+    this.set('dispositionForAllActions', DispositionForAllActions.create());
+    this.set('modalOpen', false);
```

Comment:
`this.setProperties({ ... });`

Reply:
oh yeah, originally did setProperties but thought this would be cleaner to get back to default values